src/Infrastructure.Styles.Analyzer.Tests/RMSYN1104OperatorChainingWithNewlineAnalyzerTests.cs

```diff
@@ -0,0 +1,455 @@
+﻿// This file is part of the re-motion Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// re-motion Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+
+using Gu.Roslyn.Asserts;
+using NUnit.Framework;
+using StyleAnalyzer = Infrastructure.Styles.Analyzer.RMSYN1104OperatorChainingWithNewlineAnalyzer;
+
+namespace Infrastructure.Styles.Analyzer.Tests
+{
+  [TestFixture]
+  public class RMSYN1104OperatorChainingWithNewlineAnalyzerTests
+  {
+    private StyleAnalyzer Analyzer = new StyleAnalyzer();
+
+    [Test]
+    [TestCase("||")]
+    [TestCase("&&")]
+    [TestCase("^")]
+    [TestCase("&")]
+    [TestCase("|")]
```

Comment:
I think readability of the tests would take a big hit. As they represent the spec of the analyzer I think it would be more beneficial to leave them like this.

Reply:
Then we leave it as is